Sources/ClientRuntime/Networking/Http/HttpContent.swift

```diff
@@ -0,0 +1,102 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AwsCommonRuntimeKit
+
+/// A class that implements the `IStreamable` protocol for `HttpBody`.
+/// It acts as a bridge between AWS SDK and CRT.
+class HttpContent: IStreamable {
```

Comment:
Something like `HttpBodyStreamable` is a bit more clear on the intention and removes the synonymous naming we have currently, where Body and Content have basically the same meaning.

Alternatively, there's also doing something like `AnyStreamable<T>` where the initiailizer takes closures for each of the IStreamable functions. This makes it really simply to compose anything into a streamable. Likely overkill rn and so i prefer simply changing the name.

Reply:
`StreamableHttpBody` ?

Streamable suffix sounds odd.

Reply:
done